packages/coil-extension/src/background/services/Stream.ts

```diff
@@ -121,9 +129,9 @@ export class Stream extends EventEmitter {
     this._requestId = requestId
     this._spspUrl = spspEndpoint
     this._authToken = token
-    this._tiers = container.get(BandwidthTiers)
     this._coilDomain = container.get(tokens.CoilDomain)
     this._anonTokens = container.get(AnonymousTokens)
+    this._schedule = container.get(PaymentScheduler)
```

Comment:
`container.get(PaymentScheduler)` gets the same `PaymentScheduler` for every `Stream` instance instead of a different one every time. @sublimator do you know how to get `inversify` to make a new one?

Reply:
@sentientwaffle 
When configuring the container:
> container.bind(PaymentScheduler).toSelf().inTransientScope()

Reply:
Thanks!